Cargo.toml

```diff
@@ -21,6 +21,9 @@ path = "src/main.rs"
 # This has to line up with the workspace version above
 steel-core = { path = "./crates/steel-core", version = "0.6.0", features = ["dylibs", "markdown", "stacker", "dylib-build"] }
```

Comment:
I figured you would need to add it here, so that the application uses it

Reply:
Ah, it was unclear to me whether you wanted this enabled by default or not, that's why I exposed it as a top-level feature for the main binary. If you want it enabled by default, maybe it would make sense to add it to `default-features` so that consumers can still disable it?

Reply:
Ah right, that makes sense, thanks for the clarification. For now lets leave it as is, and once we're done with the benchmarks we can decide if its default or not